docs/content/user-guides/cert-manager.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Traefik Cert Manager Integration"
```

Comment:
Here I would like to know if the product name should not be Cert-Manager (same for all occurrences)? It is branded like this in their documentation: https://cert-manager.io/docs/

Reply:
It should be `cert-manager` see https://cert-manager.io/docs/faq/#how-to-write-cert-manager

Reply:
Thanks. Fixed with 29e346b05